velox/exec/OutputBuffer.h

```diff
@@ -74,6 +74,36 @@ class ArbitraryBuffer {
   std::deque<std::shared_ptr<SerializedPage>> pages_;
 };
 
+/// The stats of a DestinationBuffer. The data transferred by the buffer has
+/// two phases:
+/// 1. Buffered: after added via enqueue() and before acked, the data is
```

Comment:
Notice, the use of 2 terms, add and enqueue, for the same thing. Let's use "enqueue".

recordEnqueue

Reply:
done.